src/status_im/ui2/screens/chat/photo_selector/view.cljs

```diff
@@ -0,0 +1,107 @@
+(ns status-im.ui2.screens.chat.photo-selector.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo.components.safe-area :as safe-area]
+            [quo2.core :as quo2]
+            [i18n.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [quo2.components.notifications.info-count :as info-count]
+            [react-native.linear-gradient :as linear-gradient]
+            [status-im.ui2.screens.chat.photo-selector.style :as style]))
+
+(def selected (reagent/atom []))
+
+(defn small-image [item]
+  [rn/view
+   [rn/image {:source {:uri item}
+              :style  {:width         56
+                       :height        56
+                       :border-radius 8
+                       :margin-bottom 20}}]
+   [rn/touchable-opacity
+    {:on-press (fn [] (reset! selected (vec (filter #(-> % (not= item)) @selected))))
```

Comment:
```clj (vec (filter ```
Can be `filterv`

Reply:
I don't think `(-> )` is necessary since we're threading once

Reply:
My recommendation here is very similar to this one: https://github.com/status-im/status-mobile/pull/14427#discussion_r1030264552

Also, `filterv` exists if you want the result as a vector. But can't it just be a list? Do you need efficient insertion at the end? Otherwise it could probably just be a list.